File: Chapter 03/Python/3.3.2 OCP/Before/midfielder.py
class Midfielder:
    def execute_play(self, play_name):
        if play_name == "DribblePastOpponent":
            print("  [Action] Executing a dribble move…")
        elif play_name == "DefensiveFormation":
            print("  [Action] Getting into defensive position…")
        else:
            print(f"  [Error] Unknown play: {play_name}")

File: Chapter 03/C#/3.3.2 OCP/Before/Midfielder.cs
using System;

namespace Chapter03.OCP.Before;

public class Midfielder
{
    public void ExecutePlay(string playName)
    {
        if (playName == "DribblePastOpponent") 
        {
            Console.WriteLine("  [Action] Executing a dribble move…");
        }
        else if (playName == "DefensiveFormation")
        {
            Console.WriteLine("  [Action] Getting into defensive position…");
        }
        else 
        {
            Console.WriteLine($"  [Error] Unknown play: {playName}");
        }
    }
}